sklearn/covariance/outlier_detection.py

```diff
@@ -62,9 +69,16 @@ def decision_function(self, X, raw_values=False):
             such as the One-Class SVM.
 
         """
+
+        if raw_values is not None:
+            warnings.warn(
+                'raw_values keyword has been moved to class initialization',
+                DeprecationWarning)
+            self.raw_values = raw_values
+
         check_is_fitted(self, 'threshold_')
         mahal_dist = self.mahalanobis(X)
-        if raw_values:
+        if self.raw_values:
             decision = mahal_dist
         else:
```

Comment:
well we need to rename the parameter at the class level anyway, so this isn't going to be as inconsistent as it sounds.